index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio - ZC</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./icons/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <!-- Toolbar -->
+    <div class="toolbar">
+        <p id="toolbartext">Welcome</p>
+        <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+    </div>
+
+    <!-- Headline Section-->
+    <div class="headline">
+
+        <div class="container">
+            <h1>Hey there. I'm Cristian<br><span style="color: #36B37E"> I'm a software developer</span> </h1>
+        </div>
+
+        <div class="container">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+
+        <div class="container">
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/twitter.svg" alt="Twitter">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/linkedinicon.svg" alt="LinkedIn">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/medium.svg" alt="Medium">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/github.svg" alt="GitHub">
+            </div>
+
+            <div class="socialmedia">
+                <img src="./icons/socialmedia/angellist.svg" alt="Angel List">
+            </div>
+           
+        </div>
+
```

Comment:
- [ ] Kindly 😃 try to add the icon here in the first section to make the user in the future when clicking on it push him/her to the second section, so it will be nice for the user to see that in the portfolio 😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/219169034-601d68ff-6f12-4176-842c-27cd38dad678.png)   |    ![image](https://user-images.githubusercontent.com/83360636/219169123-20c189d6-c6bb-465b-91d9-f224ff6d8e8c.png) |